content/docs/operating/configuration.md

```diff
@@ -546,7 +546,7 @@ Where `<filename_pattern>` may be a path ending in `.json`, `.yml` or `.yaml`. T
 may contain a single `*` that matches any character sequence, e.g. `my/path/tg_*.json`.
 
 
-### `<relabel_config>`
+### `<relabel_configs>`
```

Comment:
This is singular, as it's only about one `relabel_config` rather than a list of them.